cg/cli/set.py

```diff
@@ -230,25 +200,20 @@ def sample(
                     )
                 else:
                     comment += _generate_comment(
-                        "Application tag",
-                        sample_obj.application_version_id,
-                        application_version_id,
+                        "Application tag", sample_obj.application_version_id, application_version_id
                     )
                     sample_obj.application_version_id = application_version_id
                     echo_msg += click.style(
-                        f"\nApplication tag set to {str(application_version)}.",
-                        fg="green",
+                        f"\nApplication tag set to {str(application_version)}.", fg="green"
                     )
 
         if not skip_lims:
             context.obj["lims"].update_sample(sample_id, application=apptag)
             click.echo(click.style(f"Set LIMS/application to {apptag}", fg="blue"))
 
-    if capture_kit:
+    if capture_kit is not None:
```

Comment:
This is the only change right? 👍

Reply:
Better this way, I saw many errors due to the previous code!

Reply:
right